ycms/static/src/js/ward/discharge.ts

```diff
@@ -0,0 +1,103 @@
+import { Modal, ModalOptions } from "flowbite";
+
+let selectedBedId: string | null = "";
+let selectedBedAssignmentId: string | null = "";
+
+export const handleModalClose = (): void => {
+    document.getElementById("cancelDischargeButton")?.addEventListener("click", () => {
+        const modal = (FlowbiteInstances as any).getInstance("Modal", "modalEl");
+        modal.hide();
+    });
+    document.getElementById("closeModalButton")?.addEventListener("click", () => {
+        const modal = (FlowbiteInstances as any).getInstance("Modal", "modalEl");
+        modal.hide();
+    });
+};
+
+export const handleConfirmDischarge = (): void => {
+    document.getElementById("confirmDischargeButton")?.addEventListener("click", () => {
+        const getCookie = (name: string): string | null => {
+            let cookieValue = null;
+            if (document.cookie && document.cookie !== "") {
+                const cookies = document.cookie.split(";");
+                for (let i = 0; i < cookies.length; i++) {
+                    const cookie = cookies[i].trim();
+                    // Check if the cookie name matches the provided name
+                    if (cookie.substring(0, name.length + 1) === `${name}=`) {
+                        cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+                        break;
+                    }
+                }
+            }
+            return cookieValue;
+        };
+
+        // Use AJAX to mark the patient as discharged
+        fetch(`discharge-patient/${selectedBedAssignmentId}/${selectedBedId}/`, {
```

Comment:
```suggestion
        fetch(`/discharge-patient/${selectedBedAssignmentId}/`, {
```

since the other information is no longer needed (see above)